sources.cfg

```diff
@@ -9,3 +9,9 @@ auto-checkout =
     ftw.simplelayout
 # https://github.com/4teamwork/ftw.datepicker/pull/3
     ftw.datepicker
+# https://github.com/4teamwork/ftw.testing/pull/21
+    ftw.testing
+
+
+[branches]
+ftw.testing = jone-forbiddenfruit
```

Comment:
Cool, thank you @jone.

I've updated this pull request. The extras `test` now requires ftw.testing >= 1.10.4.